### PR TITLE
Use new browser tools and improve authentication mechanism

### PR DESCRIPTION
**Pull request by AndreyBelym**

### CODE REVIEW

**Comment by AndreyBelym** [comment]
@VasilyStrelyaev, could you please help with this information message:

https://github.com/DevExpress/testcafe/pull/4424/files#diff-2f06a6524ca61c70f30dd99f9aa083dcR1

File: src/notifications/information-message.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import dedent from 'dedent';
+
+
+export const SCREEN_RECORDING_PERMISSION_REQUEST = dedent `
```
Comment (VasilyStrelyaev) [inline src/notifications/information-message.js]: Version approved by Dirk is:

TestCafe requires permission to record the screen. Open 'System Preferences > Security & Privacy > Privacy > Screen Recording' and check 'TestCafe' in the application list.

Press any key to retry.

Comment (AndreyBelym) [reply src/notifications/information-message.js]: Yeah, right, my mistake

Comment (AndreyBelym) [reply src/notifications/information-message.js]: Yeah, right, my mistake 😢

**Pull request merged**